crates/harness-tests/src/routing.rs

```diff
@@ -199,3 +200,29 @@ async fn monolith_double_load_room(ctx: &mut TestRunner) {
     let t = resp.text().await.expect("failed to read http response");
     assert_eq!(t, "{\"name\":\"foo\"}");
 }
+
+#[test_context(TestRunner)]
+#[tokio::test]
+async fn unicast_messaging(ctx: &mut TestRunner) {
+    let mut m = Monolith::new(ctx).await.unwrap();
```

Comment:
You'll need to opt in to the client tracking monolith behavior: `BehaviorTrackClients`. Use `MonolithBuilder`.